boa_ast/src/keyword/tests.rs

```diff
@@ -0,0 +1,354 @@
+            Sym::FALSE => assert_eq!(k, Keyword::False),
+            Sym::FINALLY => assert_eq!(k, Keyword::Finally),
+            Sym::FOR => assert_eq!(k, Keyword::For),
+            Sym::FUNCTION => assert_eq!(k, Keyword::Function),
+            Sym::IF => assert_eq!(k, Keyword::If),
+            Sym::IN => assert_eq!(k, Keyword::In),
+            Sym::INSTANCEOF => assert_eq!(k, Keyword::InstanceOf),
+            Sym::IMPORT => assert_eq!(k, Keyword::Import),
+            Sym::LET => assert_eq!(k, Keyword::Let),
+            Sym::NEW => assert_eq!(k, Keyword::New),
+            Sym::NULL => assert_eq!(k, Keyword::Null),
+            Sym::OF => assert_eq!(k, Keyword::Of),
+            Sym::RETURN => assert_eq!(k, Keyword::Return),
+            Sym::SUPER => assert_eq!(k, Keyword::Super),
+            Sym::SWITCH => assert_eq!(k, Keyword::Switch),
+            Sym::THIS => assert_eq!(k, Keyword::This),
+            Sym::THROW => assert_eq!(k, Keyword::Throw),
+            Sym::TRUE => assert_eq!(k, Keyword::True),
+            Sym::TRY => assert_eq!(k, Keyword::Try),
+            Sym::TYPEOF => assert_eq!(k, Keyword::TypeOf),
+            Sym::VAR => assert_eq!(k, Keyword::Var),
+            Sym::VOID => assert_eq!(k, Keyword::Void),
+            Sym::WHILE => assert_eq!(k, Keyword::While),
+            Sym::WITH => assert_eq!(k, Keyword::With),
+            Sym::YIELD => assert_eq!(k, Keyword::Yield),
+            _ => unreachable!("unknown keyword {k:?} found"),
+        }
+    }
+}
+
+#[test]
+fn ut_try_into_binary_op() {
+    for k in all_keywords() {
+        match k {
+            Keyword::InstanceOf | Keyword::In => assert!(BinaryOp::try_from(k).is_ok()),
+            Keyword::Await
+            | Keyword::Async
+            | Keyword::Break
+            | Keyword::Case
+            | Keyword::Catch
+            | Keyword::Class
+            | Keyword::Continue
+            | Keyword::Const
+            | Keyword::Debugger
+            | Keyword::Default
+            | Keyword::Delete
+            | Keyword::Do
+            | Keyword::Else
+            | Keyword::Enum
+            | Keyword::Export
+            | Keyword::Extends
+            | Keyword::False
+            | Keyword::Finally
+            | Keyword::For
+            | Keyword::Function
+            | Keyword::If
+            | Keyword::Import
+            | Keyword::Let
+            | Keyword::New
+            | Keyword::Null
+            | Keyword::Of
+            | Keyword::Return
+            | Keyword::Super
+            | Keyword::Switch
+            | Keyword::This
+            | Keyword::Throw
+            | Keyword::True
+            | Keyword::Try
+            | Keyword::TypeOf
+            | Keyword::Var
+            | Keyword::Void
+            | Keyword::While
+            | Keyword::With
+            | Keyword::Yield => assert!(BinaryOp::try_from(k).is_err()),
+        }
+    }
+}
+
+#[test]
+fn ut_from_str() {
```

Comment:
What does `ut` mean?

Reply:
haha "unit test". I tend to prefix unit tests with `ut_`, and integration tests at `it_`. I think we have not done this in Boa until now, so I can remove it if needed, but might make sense.

Reply:
To me it seems unneeded, because in rust tests that are defined in the source are unit tests, and tests that are defined externally in the [`tests` dir in root crate level](https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html) are integration tests.

Reply:
While it's clear during development, it might not be so much when running them.

But I get it xD I will just remove it :)

Reply:
Prefix removed